CRUD.c

```diff
@@ -0,0 +1,186 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define FILENAME "users.txt"
+
+// User structure
+typedef struct {
+    int id;
+    char name[40];
+    int age;
+} User;
+
+// Let's check whether the file exists or not
+void checkFile() {
+    FILE *file = fopen(FILENAME, "a");
+    if (file == NULL) {
+        printf("Something's wrong.\n");
+        exit(1);
+    }
+    fclose(file);
+}
+
+// Create a new user
+void createUser() {

```

Comment:
The requirement was to have a unique user ID. The code doesn't seem to consider and fulfill this requirement.